docs/__tests__/inline-filter.test.tsx

```diff
@@ -0,0 +1,135 @@
+import { render } from '@testing-library/react';
```

Comment:
Let's colocate this file/directory as a sibling to `InlineFilter` and use PascalCase for the test file name, e.g.
```
components/
  InlineFilter.tsx
  __tests__/
    InlineFilter.test.tsx
```